src/api/fs/watching.ts

```diff
@@ -0,0 +1,386 @@
+import { ChangeSet, ChangeSpec, Text } from "@codemirror/state";
+import { extensionPort, proxy } from "src/util/comlink";
+import { TextChange, WatchTextFileListeners } from "src/types";
+
+/**
+ * A helper to change a ChangeSet into a simpler serializable & human readable format
+ */
+function changeSetToSimpleTextChange(changes: ChangeSet): Array<TextChange> {
```

Comment:
Let's make sure we import `ChangeSet` in `types/index.ts` and then export it so that it gets documented in Typedoc.  Let's also be sure to add the type to `apis.json`

Reply:
ChangeSet is not visible to users, I removed it from the API